src/config.zig

```diff
@@ -77,7 +77,7 @@ const ConfigProcess = struct {
     verify: bool,
     port: u16 = 3001,
     address: []const u8 = "127.0.0.1",
-    storage_size_max: u64 = 16 * 1024 * 1024 * 1024 * 1024,
+    storage_size_limit: u64 = 16 * 1024 * 1024 * 1024 * 1024,
```

Comment:
Maybe this should be `storage_size_limit_default`, to disambiguate it from the "real" runtime storage limit (via CLI arg)?

Reply:
I think it is `limit_max` rather than `limit_default`. Or rather, it is both default and max, but the max is more salient of the two